__tests__/app.test.js

```diff
@@ -203,7 +207,59 @@ describe("GET /api/articles", () => {
       expect(articles).toBeSortedBy('author')
     })
   })
+  })
+  describe("?topic", () => {
+    test("200: accepts topic query which filters the articles by the topic value specified in the query", () => {
+      return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({body: { articles }}) => {
+        expect(articles).toHaveLength(1)
+        expect(articles[0]).toMatchObject({
+          article_id: expect.any(Number),
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: 0,
+          article_img_url:"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: expect.any(Number)
+        })
```

Comment:
Rather than checking all of the first article, you could loop over all of the articles and just ensure the topic for each one is "cats" 🙂